_layouts/usecases.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+---
+<div class="card gx-5 gy-5 m-5">
+    <div class="card-header">
+        <div class="card-title display-6 mx-2 mt-3 mb-4">Use Cases</div>
+        <ul class="nav nav-tabs nav-justified fs-5 card-header-tabs" id="usecases" role="tablist">
+            <li class="nav-item">
+                <button class="nav-link active" id="encryption-at-rest-usecase" data-bs-toggle="tab" data-bs-target="#encryption-at-rest-usecase-tab-pane" type="button" role="tab" aria-controls="encryption-at-rest-usecase-tab-pane" aria-selected="true">
```

Comment:
I think from reading the [Jekyll docs](https://jekyllrb.com/docs/posts/) we should be able to auto generate the tabs from the directory content but I'm not exactly sure what that would look like in practice. 

Certainly something for a separate PR.

Reply:
As above, I think https://jekyllrb.com/docs/collections/ might be the way.